Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Storage/Queries/InvoiceExtra.hs

```diff
@@ -163,7 +163,7 @@ findLatestNonAutopayActiveByDriverId driverId serviceName = do
   findAllWithOptionsKV
     [ Se.And
         [ Se.Is BeamI.driverId $ Se.Eq (getId driverId),
-          Se.Is BeamI.paymentMode $ Se.Not $ Se.Eq Domain.AUTOPAY_INVOICE,
+          Se.Is BeamI.paymentMode $ Se.Not $ Se.In [Domain.AUTOPAY_INVOICE, Domain.PAYOUT_REGISTRATION_INVOICE, Domain.ONE_TIME_SECURITY_INVOICE],
```

Comment:
Any impact on subscription?

Reply:
in subscription, we were fetching the latest invoice, that are not part of AUTOPAY_INVOICE, but now this PAYOUT_REGISTRATION_INVOICE will also be there for this payout case , which will come via this query, to hanlde that, this change is required

Reply:
@hkmangla will do scrutiny of queries for driver fee and invoice. above doesn't seem to have an impact and the above is blocking further testing.